src/components/PrimaryNav.tsx

```diff
@@ -96,6 +97,21 @@ const PrimaryNav = (): JSX.Element => {
                   )}
                 </Menu.Item>
 
+                <Menu.Item>
+                  {({active}) => (
+                    <button
+                      onClick={async () => {
+                        window.open(bugReportLink);
```

Comment:
<a href={bugReportLink} />
  <button></button>
</a>

interesting use of window.open. Curious if there is value over using a href?

Reply:
I suppose less lines and use of button onClick may be more accesible?